app/javascript/mastodon/reducers/compose.js

```diff
@@ -223,7 +223,12 @@ const hydrate = (state, hydratedState) => {
   state = clearAll(state.merge(hydratedState));
 
   if (hydratedState.has('text')) {
-    state = state.set('text', hydratedState.get('text'));
+    const text = hydratedState.get('text');
+    if (text) {
+      state = state
+        .set('text', hydratedState.get('text'))
+        .set('focusDate', new Date());
+    }
```

Comment:
Oh that's actually a good way to do it!

Code-style wise, I'd do something like this though:
```suggestion
  if (hydratedState.get('text')) {
    state = state.set('text', hydratedState.get('text')).set('focusDate', new Date());
  }
```

Reply:
Thanks for the advice. I've pushed the commit again :)